index.html

```diff
@@ -2331,7 +2321,7 @@ <h3>Presentations</h3>
 The example below shows a [=verifiable presentation=]:
         </p>
 
-        <pre class="example nohighlight" title="Basic structure of a presentation">
+        <pre class="example nohighlight" title="Basic structure of a presentation secured with Data Integrity">
```

Comment:
I do not think it is necessary for this example. Example 18 does not use _any_ proof mechanisms, so the reference to DI looks irrelevant.